dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -45,6 +60,14 @@ const findCollectionSuitableForFrontBranding = (
 	return index;
 };
 
+const getContainerLevel = (
+	levels?: FEContainerPalette[],
```

Comment:
Currently, the level is included in the config meta from frontend. This is largely only used for container palette which is why the type here might seem strange. 

There are 3 options here:
1. Leave as is with the comment 
2. Rename FEContainerPalette to be more generic and better represent the metadata.
3. Separate out the container level tags from the container palette tags on frontend model. There is a spike here for this work